app/controllers/conversations_controller.rb

```diff
@@ -37,7 +36,12 @@ def create
     # This will have to be removed when mobile autocomplete is ported to Typeahead
     recipients_param, column = [%i(contact_ids id), %i(person_ids person_id)].find {|param, _| params[param].present? }
     if recipients_param
-      person_ids = current_user.contacts.mutual.where(column => params[recipients_param].split(",")).pluck(:person_id)
+      # As an admin, I want to send a message to all local user
```

Comment:
Okay now I get why you modified the contacts controller, it was to get the autocomplete here to work. But as an admin is able to contact anyone, I guess to add a new route which returns every local user is fine, and this shouldn't be in the contacts controller I guess. Maybe it even already exists. Can you tell us @jhass, or where to put it if it doesn't exist?

Reply:
Function differs if admin parses person_ids or a non-admin user parses person_ids. Different endpoints seems for me a bit complicated. (Check in frontend which role a user has, then change endpoints for same data with different filter) What do others think?